Aspire.Pizzeria.PizzaStore.Contracts/PlacedOrderCommand.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace Aspire.Pizzeria.PizzaStore.Contracts;
+
+public record PlacedOrderCommand(string CustomerName, string DeliveryAddress, int[] PizzaIds);
```

Comment:
This should probably be moved into the the Pizza Store service project and we should have a request class/record here